arch/xtensa/src/common/xtensa_context.S

```diff
@@ -3,12 +3,12 @@
  *
  * Adapted from use in NuttX by:
  *
- *   Copyright (C) 2016 Gregory Nutt. All rights reserved.
- *   Author: Gregory Nutt <gnutt@nuttx.org>
+ *	 Copyright (C) 2016 Gregory Nutt. All rights reserved.
```

Comment:
Please get back spaces in comments

Reply:
This should've been reverted for both file now.